docs/source/jupyter/ontology_interface.ipynb

```diff
@@ -201,6 +201,28 @@
     "- When the keyword `reference_by_label` is set to `False` (disabled) or not set, the dot notation is a shorthand for fetching by suffix instead. This keyword is **disabled** in the `city` namespace."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"admonition important\">\n",
+    "\n",
+    "The dot notation supports IPython autocompletion. For example, when working on a Jupyter notebook, once the namespace has been imported, it is possible to get suggestions for the entity names by writing `namespace.` and pressing TAB. \n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"admonition important\">\n",
+    "\n",
+    "To get a list of all the entities available within a namespace, run `list(namespace)`. \n",
```

Comment:
I suggest moving up this line, to just before the sentence "There are several ways to access an ontology entity in OSP-core".

Also, I don't see why it should be highlighted with a green background.